docs/source/ops/index.rst

```diff
@@ -0,0 +1,86 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. ops-ref_index:
+
+DAML Participant pruning
+========================
+
+.. HINT::
+   DAML Participant pruning is currently an :doc:`Early Access Feature in Labs status </support/status-definitions>`
+
+The DAML Ledger API exposes an append-only ledger model; on the other hand, DAML Participants must be able to operate continuously for an indefinite amount of time on a limited amount of hot storage.
+
+In addition, privacy demands [1]_ may require removing Personally Identifiable Information (PII) upon request.
+
+To satisfy these requirements, the :ref:`Pruning Service <com.daml.ledger.api.v1.admin.ParticipantPruningService>` Ledger API endpoint [2]_ allows DAML Participants to support pruning of DAML contracts and transactions that were respectively archived and submitted before or at a given ledger offset.
+
+Please refer to the specific DAML Driver information for details about its pruning support.
+
+.. [1] For example, as enabled by provisions about the "right to be forgotten" of legislation such as
+       `EU's GDPR <https://gdpr-info.eu/>`_.
+.. [2] Invoking the Pruning Service requires administrative privileges.
+
+Impacts on DAML applications
+----------------------------
+
+When supported, pruning can be invoked by an operator with administrative privileges at any time on a healthy DAML participant; furthermore, it doesn't require stopping nor suspending normal operation.
+
+Still, DAML applications may be affected in the following ways:
+
+- Pruning is potentially a long-running operation and demanding one in terms of system resources; as such, it may significantly reduce DAML Ledger API throughput and increase latency while it is being performed. It is thus recommended to plan pruning invocations when low system utilization is expected.
+- Pruning may degrade the behavior of and/or crash the system if the pruning offset is too recent. In particular, the system might misbehave if command completions are pruned before the command trackers are able to process the completions.
+- Pruning may affect the behavior of Ledger API calls that allow to read data from the ledger: see the next sub-section for more information about API impacts.
+
+How the DAML Ledger API is affected
+-----------------------------------
+
+- Active data streams from the DAML Participant may abort and need to be re-established by the DAML application from a later offset than pruned, even if they are already streaming past it.
```

Comment:
Regarding "even if they are already streaming past it", I believe with the current code, this is only the case in the `CommandCompletions` case, but nevertheless for clarity it is probably good to leave this sentence in here also in case we later find it becomes necessary to abort transaction streams more aggressively too.

Reply:
Yes, I'd rather stay a bit more vague here and detail these aspects in the API docs, as in this way we won't need to update it if details change (esp. for the worse). They are less relevant for the operator anyway, actually we might also consider removing the API-related sections altogether from the ops guide. WDYT @gerolf-da ?

Reply:
The operator needs to interact with the API to trigger pruning, so I'm fine with having a more vague representation of the API here, together with a link to the API docs.